.travis.yml

```diff
@@ -164,7 +164,8 @@ matrix:
         - docker
       env: JOB_TYPE=prerelease UNDERLAY_REPOSITORY_NAMES="roscpp_core" OVERLAY_PACKAGE_NAMES=roscpp
       before_script:
-        - pip3 install EmPy
+        - sudo apt-get install -y python3-pip
```

Comment:
This needs to call `apt-get update` before in order to reliably install the package.